src/Service/OrderShipper.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Boekuwzending\Magento\Service;
+
+use Boekuwzending\Magento\Api\OrderRepositoryInterface;
+use Boekuwzending\Magento\Utils\Constants;
+use Exception;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NotFoundException;
+use Magento\Sales\Model\Convert\Order as ConvertOrder;
+use Magento\Sales\Model\Order\Shipment;
+use Magento\Sales\Model\Order\Shipment\TrackFactory;
+use Magento\Shipping\Model\ShipmentNotifier;
+use Magento\Sales\Api\OrderRepositoryInterface as MagentoOrderRepositoryInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Class OrderShipper
+ *
+ * Gets called by the LabelCreated webhook to ship an order and apply tracking codes (labels) to the shipment.
+ *
+ * @package Boekuwzending\Magento\Service
+ */
+class OrderShipper implements OrderShipperInterface
+{
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+    /**
+     * @var OrderRepositoryInterface
+     */
+    private $orderRepository;
+    /**
+     * @var MagentoOrderRepositoryInterface
+     */
+    private $magentoOrderRepository;
+    /**
+     * @var ConvertOrder
+     */
+    private $orderConverter;
+    /**
+     * @var TrackFactory
+     */
+    private $trackFactory;
+    /**
+     * @var ShipmentNotifier
+     */
+    private $shipmentNotifier;
+    /**
+     * @var ShipmentCommentCreationInterface
+     */
+    private $commentCreation;
+
+    /**
+     * OrderShipper constructor.
+     * @param LoggerInterface $logger
+     * @param OrderRepositoryInterface $orderRepository
+     * @param MagentoOrderRepositoryInterface $magentoOrderRepository
+     * @param ConvertOrder $orderConverter
+     * @param TrackFactory $trackFactory
+     * @param ShipmentNotifier $shipmentNotifier
+     */
+    public function __construct(LoggerInterface $logger,
+                                OrderRepositoryInterface $orderRepository,
+                                MagentoOrderRepositoryInterface $magentoOrderRepository,
+                                ConvertOrder $orderConverter,
+                                TrackFactory $trackFactory,
+                                ShipmentNotifier $shipmentNotifier)
```

Comment:
is dit std magento? ander zou ik zeggen alles 1 regel naar beneden en naar voren uitlijnen